src/cosmos-keystore.ts

```diff
@@ -103,13 +98,13 @@ function loadFromStorage(address: string): StoredWallet | null {
 }
 
 // stores an encrypted wallet in localstorage
-function addToStorage(name: string, address: string, ciphertext: string, network: string): void {
-  addToIndex(name, address)
+function addToStorage(name: string, address: string, wallet: string, network: string): void {
+  addToIndex(name, address, wallet)
```

Comment:
the index doesn't have the wallet on purpose to not have double data maintenance

Reply:
we don't add the wallet to the index